### ISSUE TITLE
Errors related to possible validation of Geospatial Bounding Box fields prevents or slows dataset publishing

### ISSUE THREAD

**Issue by jggautier** [IQSS/dataverse#9421]
**What steps does it take to reproduce the issue?**

* When does this issue occur?
When the trying to create a dataset version using the "Create a dataset in a Dataverse collection" and "Import a Dataset into a Dataverse Collection" endpoints and the JSON file includes "invalid" data in Geospatial Bounding Box fields.

  And when using a web browser to try to create or edit a dataset version that includes "invalid" data in the Geospatial Bounding Box fields.

* Which page(s) does it occurs on?
The dataset create and edit pages.

* What happens?
The dataset version can't be created or can't be saved.

  When using the API endpoints:
  I've seen two errors messages: one error when there are non-numerical values in any of the Geospatial Bounding Box fields, and another error when the number in the North Latitude field is less than the number in the South Latitude field.

  When using a web browser:
  After clicking Save Changes I get the error "Error – The metadata could not be updated. If you believe this is an error, please contact Demo Dataverse Support for assistance." but I can see that the metadata was updated.

  The Publish button doesn't work until you refresh the page, which removes the error message. Then I can publish the dataset.

  But the dataset URL includes "&version=DRAFT" and the page shows the Info banner that reads that a draft version wasn't found. This might be related to an already documented bug.

* To whom does it occur (all users, curators, superusers)?
Superusers, curators, depositors

* What did you expect to happen?
Has validation been added to the Geospatial Bounding Box fields? If it has, the error message in the GUI should make clear what's preventing the dataset version from being saved. The API endpoints return more specific error messages, but I'm not sure if they are clear to others.

**Which version of Dataverse are you using?**
v5.13

**Comment by jggautier** [comment]
In Dataverse installations running v5.13, this "invalid" bounding box metadata will prevent datasets from being updated.

An example in the Harvard Dataverse Repository is the dataset at https://doi.org/10.7910/DVN/1HTKPJ.

As other installations update their Dataverse software to v5.13, their datasets with this "invalid" metadata also won't be able to be updated.

An example in HeiDATA's installation is the dataset at https://doi.org/10.11588/data/LSG8TN.

People who try to update datasets like these will see an error message with no information about what metadata could not be updated:
<kbd><img width="1159" alt="Screenshot 2023-03-09 at 12 54 33 PM" src="https://user-images.githubusercontent.com/18374574/224113922-58d7f754-ffad-44a1-9427-2e52803a1adf.png"></kbd>

**Comment by pdurbin** [comment]
> maybe as part of the Geospatial Search added in v5.13

Yes, I suspect this is a Solr error and due to the addition of geospatial search.

What should we do with bounding box data? Save it anyway like we always have, and not index it into Solr?

Or should the user get a validation error "lat/long wrong blah blah" that prevents them from saving, similar to an invalid URL or email address?

**Comment by mreekie** [comment]
grooming
- Looking at the NetCDF deliverables.
- This looks like it's a related issue that is related to active work, but it is not in the netcdf backlog Queue
- Not going to add it to the queue, but tagging it.

**Comment by meghangoodchild** [comment]
Just wanted to add that we experienced this issue at Borealis due to invalid metadata in the geographic bounding box fields, which prevented proper indexing of the dataset. 
It would be helpful if there was a validation error that points to the issue. It took us awhile to figure it out, and we also had trouble republishing, even after removing the invalid metadata.
Thanks!

**Comment by pdurbin** [comment]
@meghangoodchild yikes. Thanks for the heads up. I'm not surprised that installations are starting to see this problem.

Like @jggautier noted, the following issue is related, and we could probably collapse it into one effort:

- #9547

Also, I was poking at #9593 the other day and ran into an indexing problem: https://github.com/IQSS/dataverse/pull/9595#issuecomment-1551666633

It would be nice if any fix we put it ultimately sends only the numerical values to Solr (after validation, of course), to unblock that issue and pull request.

**Comment by cmbz** [comment]
2023/10/02: Depending upon sizing, if small add to 6.1, otherwise we can defer. There is a workaround, if necessary.

**Comment by cmbz** [comment]
**2023/10/04**
- Note that this issue is being closed because it will be addressed more thoroughly by https://github.com/IQSS/dataverse/issues/9547
